packages/instructions/src/accounts.ts

```diff
@@ -2,33 +2,90 @@ import { Address } from '@solana/addresses';
 
 import { AccountRole } from './roles';
 
+/**
+ * Represents an account's address and metadata about its mutability and whether it must be a signer
+ * of the transaction.
+ *
+ * Typically, you will use one of its subtypes.
+ *
+ * |                                   | `role`                        | `isSigner` | `isWritable` |
+ * | --------------------------------- | ----------------------------- | ---------- | ------------ |
+ * | `ReadonlyAccount<TAddress>`       | `AccountRole.READONLY`        | &#x274c;   | &#x274c;     |
+ * | `WritableAccount<TAddress>`       | `AccountRole.WRITABLE`        | &#x274c;   | &#x2705;     |
+ * | `ReadonlySignerAccount<TAddress>` | `AccountRole.READONLY_SIGNER` | &#x2705;   | &#x274c;     |
+ * | `WritableSignerAccount<TAddress>` | `AccountRole.WRITABLE_SIGNER` | &#x2705;   | &#x2705;     |
```

Comment:
nit: Should we just use characters like `✔` and `𐄂` or just `true/false`? Whilst this will display properly in user-land, it's quite impractical to read and maintain in the code.

Reply:
Actually I'll just write `Yes` and `No`.

Reply:
I updated this, but only in the TypeDoc; not in the README.